include/lsst/afw/detection/Footprint.h

```diff
@@ -219,30 +161,15 @@ class Footprint : public lsst::daf::base::Citizen,
      * totally zero, and moves endpoints to non-zero; it does not
      * split spans that have internal zeros.
      */
-    template<typename PixelT>
-    void clipToNonzero(lsst::afw::image::Image<PixelT> const& img);
+     template<typename PixelT>
+     void cliptoNonZero(image::Image<PixelT> const & img);
```

Comment:
It was only used in meas_deblender. I plan on introducing the fix into meas_deblender in DM-8108